### DISCUSSION TITLE
Github student pack delaying

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22799] [New to GitHub]
<p>Hi I’m Surya from India and I applied for GitHub’s student developer pack 10 days before and still, I didn’t receive any mails please help me. :relaxed: Thank you</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>It’s the very busy season for our Education team, and I believe applications that require a manual review are taking up to 6 weeks at present.</p>
<p>I’m sorry for the wait!</p>

**Comment by ghost** [comment]
<p>Thank you for your quick reply :grinning:</p>
